index.html

```diff
@@ -30,14 +30,15 @@
 
   <div class="section slide-campaigns">
     <div class="wrapper">
-      <div class="campaign__block campaign__details">
-        <p>Staff Pick Performance</p>
-        <img alt="Staff Pick" id="campaign__image" />
+      <div class="campaign__block campaign__block-left">
```

Comment:
Is there a `campaign` element anywhere that this is a child of? If not, i'd rename `campaign-block`. Also, it looks like `campaign__block-left` is a modifier so I'd rename this `campaign-block--left` if you're going for BEM-style classes.